include/wx/qt/glcanvas.h

```diff
@@ -85,4 +87,27 @@ class WXDLLIMPEXP_GL wxGLCanvas : public wxGLCanvasBase
     wxDECLARE_CLASS(wxGLCanvas);
 };
 
+#include <QtWidgets/QGestureRecognizer>
+#include <QtWidgets/QGestureEvent>
+
+class PanGestureRecognizer : public QGestureRecognizer
```

Comment:
This doesn't seem to belong here: we really shouldn't have any non-wx private classes in public headers and we try to avoid (and, AFAICS, so far succeed) including platform/toolkit-specific headers from public wx headers.

I think this class could be just moved to `src/qt/glcanvas.cpp`, couldn't it?

Reply:
Agreed, no need to put this class in the header file as far as I can tell.  Moving it to glcanvas.cpp now.